config.json

```diff
@@ -959,6 +959,27 @@
         "nugetVersion": "1.1.0.2",
         "nugetId": "Xamarin.AndroidX.GridLayout"
       },
+      {
+        "groupId": "androidx.health.connect",
+        "artifactId": "connect-client",
+        "version": "1.1.0-rc02",
+        "nugetVersion": "1.1.0.1-rc02",
+        "nugetId": "Xamarin.AndroidX.Health.Connect.ConnectClient"
+      },
+      {
+        "groupId": "androidx.health.connect",
+        "artifactId": "connect-client-external-protobuf",
+        "version": "1.1.0-rc02",
+        "nugetVersion": "1.1.0.1-rc02",
+        "nugetId": "Xamarin.AndroidX.Health.Connect.ConnectClientExternalProtobuf"
```

Comment:
This one has many errors, can we `<remove-node/>` all the C# code? Such as:

https://github.com/dotnet/android-libraries/blob/344b3259689c5c133621c10b34513d5ea4325d20/source/com.google.guava/guava/Transforms/Transforms.xml#L3

I don't think we should try to fix the 700+ errors for this project.